src/System.Runtime.Analyzers/Core/SystemRuntimeAnalyzersResources.resx

```diff
@@ -316,10 +316,10 @@
     <value>A method or constructor calls one or more members that have overloads that accept a System.IFormatProvider parameter, and the method or constructor does not call the overload that takes the IFormatProvider parameter. When a System.Globalization.CultureInfo or IFormatProvider object is not supplied, the default value that is supplied by the overloaded member might not have the effect that you want in all locales. If the result will be based on the input from/output displayed to the user, specify 'CultureInfo.CurrentCulture' as the 'IFormatProvider'. Otherwise, if the result will be stored and accessed by software, such as when it is loaded from disk/database and when it is persisted to disk/database, specify 'CultureInfo.InvariantCulture'</value>
   </data>
   <data name="SpecifyIFormatProviderMessageIFormatProviderAlternateString" xml:space="preserve">
-    <value>Because the behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
+    <value>The behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
   </data>
   <data name="SpecifyIFormatProviderMessageIFormatProviderAlternate" xml:space="preserve">
-    <value>Because the behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
+    <value>The behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
```

Comment:
Just to explicitly state the new user experience: The detailed Description that Balaji is talking about can be seen in the error list by clicking on the ">" symbol that shows up near the shorter default error message - so the user still can view the detailed message in error list, just that it won't show up be default.